descriptors/ModuleDescriptor-template.json

```diff
@@ -904,7 +904,7 @@
       "version":"1.0"
     },
     {
-      "id": "finance-storage.job-number.get",
+      "id": "finance-storage.job-number",
```

Comment:
is it causing some failure? just interesting how error looks like if we have incorrect  interface id

Reply:
```
Missing dependency: mod-finance-5.1.0-SNAPSHOT.277 requires finance-storage.job-number.get: 1.0
[Pipeline] }
[Pipeline] // stage
```
https://jenkins-aws.indexdata.com/job/folio-org/job/mod-finance/job/master/277/console